homeassistant/components/rpi_camera/camera.py

```diff
@@ -49,6 +51,10 @@
         vol.Optional(CONF_VERTICAL_FLIP, default=DEFAULT_VERTICAL_FLIP): vol.All(
             vol.Coerce(int), vol.Range(min=0, max=1)
         ),
+        vol.Optional(CONF_OVERLAY_METADATA): vol.All(
```

Comment:
We no longer allow new integrations to use a platform config section or old integrations to do changes to the platform config section. See https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md.

Please refactor this integration to have the config schema in the base integration module and the config yaml section under the top level integration domain key.